compose-files/brizo.yml

```diff
@@ -0,0 +1,18 @@
+version: '2.1'
+services:
+  brizo:
+    image: oceanprotocol/brizo:${OCEAN_VERSION:-stable}
+    ports:
+    - 8030:8030
+    networks:
+      backend:
+        ipv4_address: 172.15.0.17
+    depends_on:
+    - keeper-contracts
+    - parity-node
+    env_file:
```

Comment:
The problem here is that neither `parity-node` nor `secret-store` exists in all environments (for example if we run with `keeper-contracts` running ganache, so these dependencies would make that brizo would not work. Also I am not fan of `depends_on` because it creates a false sense of `docker-compose` is managing any cross dependency, which is not the case, so I would prefer to delegate all that responsibility to the services themself.

Reply:
ok